interface/resources/qml/Web3DSurface.qml

```diff
@@ -43,11 +53,12 @@ Item {
             root.item = newItem
             root.item.url = url
             root.item.scriptUrl = scriptUrl
+            root.item.transparentBackground = webBackgroundColor.endsWith("FF") ? false : true
```

Comment:
Web background is always transparent unless the background has full blue in it?

Reply:
Haha, that's exactly the line of code I was going to ask about. Soooo... if the opacity is at the beginning, then that's wrong. If the opacity is at the end, then that's correct. How about I update it and see what results you get?

I use the console to test this PR. "editEntity"

Reply:
Yep that would be great. Even this shows transparency being first. https://doc.qt.io/qt-5/qml-color.html

Reply:
Indeed it does, I was following that, but when I follow that convention... it doesn't work like I expected! Maybe I'm doing it wrong (again..)